packages/core/tests/connectionless-proofs.test.ts

```diff
@@ -1,15 +1,36 @@
+import type { ProofStateChangedEvent } from '../src/modules/proofs'
+import type { SubjectMessage } from 'tests/transport/SubjectInboundTransport'
+
+import { Subject, ReplaySubject } from 'rxjs'
+
+import { SubjectInboundTransport } from '../../../tests/transport/SubjectInboundTransport'
+import { SubjectOutboundTransport } from '../../../tests/transport/SubjectOutboundTransport'
+import { Agent } from '../src/agent/Agent'
+import { Attachment, AttachmentData } from '../src/decorators/attachment/Attachment'
 import {
   PredicateType,
   ProofState,
   ProofAttributeInfo,
   AttributeFilter,
   ProofPredicateInfo,
   AutoAcceptProof,
+  ProofEventTypes,
 } from '../src/modules/proofs'
+import { LinkedAttachment } from '../src/utils/LinkedAttachment'
+import { uuid } from '../src/utils/uuid'
 
-import { setupProofsTest, waitForProofRecordSubject } from './helpers'
+import {
+  getBaseConfig,
+  issueCredential,
+  makeConnection,
+  prepareForIssuance,
+  setupProofsTest,
+  waitForProofRecordSubject,
+} from './helpers'
 import testLogger from './logger'
 
+import { CredentialPreview } from '@aries-framework/core'
```

Comment:
Maybe we should stay consistent and import `CredentialPreview` with relative path?